ChemistryLaboratory/ChemistryLaboratory/Class1.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+
+
+namespace MyApp.Class
+{
+    public class ChemistyLaboratory
+    {
+        public int maxSubstances;
+        public int currentSubstancesCount;
+        public string command;

```

Comment:
You dont need to store this field here, it should be just variable in Program.cs